app/middlewares/auth.js

```diff
@@ -0,0 +1,36 @@
+const { unauthorizedError } = require('../errors');
+const { verifyToken } = require('../helpers/auth');
+const logger = require('../logger');
+const { objIsNotEmpty } = require('../mappers/commonObjectsValidations');
+const userService = require('../services/users');
+
+exports.validateAccess = async (req, res, next) => {
+  try {
+    const token = req.headers.authorization ? req.headers.authorization.replace(/^Bearer\s+/, '') : '';
```

Comment:
I prefer to use AND and OR that ternary conditional operator 

for example 

`const token = (req.headers.authorization && req.headers.authorization.replace(/^Bearer\s+/, '')) || :  ''`